crates/notary/server/src/service/tcp.rs

```diff
@@ -84,13 +85,22 @@ pub async fn tcp_notarize(
     notary_globals: NotaryGlobals,
     session_id: String,
 ) {
+    let start = Instant::now();
     debug!(?session_id, "Upgraded to tcp connection");
     match notary_service(stream, notary_globals, &session_id).await {
         Ok(_) => {
-            info!(?session_id, "Successful notarization using tcp!");
+            info!(
+                ?session_id,
+                elapsed_time = start.elapsed().as_millis(),
```

Comment:
can we have the unit (milliseconds) printed too?

Reply:
I didn't want to include time in message so that I can later filter my message.
Therefore - changed the field name to include the units:
```
Successful notarization using tcp! session_id="..." elapsed_time_millis=0
```